plugins/builds/README.md

```diff
@@ -40,6 +40,9 @@ Arguments:
 #### Gets a build step
 `GET /builds/{id}/steps/{name}`
 
+#### Gets the current active build step
+`GET /builds/{id}/steps/active`
```

Comment:
what if a step is named `active`?

Reply:
hmm, it could get mixed. Wondering if naming it latest or current could also end up being the same, thoughts?

Reply:
we can make status as a query param. It will be difficult to pull this of this off this path param. GET /builds/{id}/steps?status=active